src/chat/__tests__/chatReducers-test.js

```diff
@@ -53,6 +53,31 @@ describe('chatReducers', () => {
       expect(newState).toEqual(expectedState);
       expect(newState).not.toBe(initialState);
     });
+
+    test('changes active narrow and anchors about message passed', () => {
+      const initialState = deepFreeze({
+        narrow: [],
+      });
+
+      const action = deepFreeze({
+        type: SWITCH_NARROW,
+        narrow: streamNarrow('some stream'),
+        anchor: { id: 1 },
+      });
+
+      const expectedState = {
+        narrow: streamNarrow('some stream'),
+        fetchingOlder: false,
+        fetchingNewer: false,
+        messages: {
+          [streamNarrowStr]: [{ id: 1 }],
+        },
+      };
+
+      const newState = chatReducers(initialState, action);
```

Comment:
Not sure the test actually tests for what it describes it does. How is the anchor tested?